smoke-tests/src/test/java/com/splunk/opentelemetry/helper/ResourceMapping.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.helper;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class ResourceMapping {
+  public static ResourceMapping of(String resourcePath, String containerPath) {
+    return new AutoValue_ResourceMapping(resourcePath, containerPath);
+  }
+
+  public abstract String resourcePath();
+
+  public abstract String containerPath();
```

Comment:
Here we are in 2021 and Java still doesn't have a reasonable `Pair` or `Tuple` type.  🙈